### PR TITLE
ss/DCOS-48183 Added GUI update page and endpoints.yaml file.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/DCOS-48183

Add GUI update pages to DC/OS 1.13 documents.

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

Added Updating the GUI page to /1.13/gui/ pages, including call to 'endpoints.yaml' API.
Created endpoints.yaml file and added text from R. Norris paper.

### CODE REVIEW

**Comment by SarahStegall** [comment]
The call to the .yaml file returns an error message. See attached. Someone needs to diagnose the error. Every change I made resulted in the same error message re: line 11.
Screen Shot 2019-03-27 at 4 10 57 PM

File: pages/1.13/api/logs.yaml [hunk 1]
```diff
@@ -3,15 +3,7 @@ swagger: '2.0'
 info:
   version: "0.0.1"
   title: DC/OS Log API v1
-  description: >
```
Comment (TattdCodeMonkey) [inline pages/1.13/api/logs.yaml]: Are we expecting this description to be removed here?

Comment (SarahStegall) [reply pages/1.13/api/logs.yaml]: I can't answer that. Can you ask Engineering?

**Comment by ck4adventure** [comment]
Commit added formats the yaml file properly.

**Comment by SarahStegall** [comment]
Thanks to @ck4adventure for your help with the swagger build!

**Pull request merged**